src/js_realm.hpp

```diff
@@ -482,14 +483,16 @@ inline typename T::Function RealmClass<T>::create_constructor(ContextType ctx) {
     FunctionType realm_constructor = ObjectWrap<T, RealmClass<T>>::create_constructor(ctx);
     FunctionType collection_constructor = ObjectWrap<T, CollectionClass<T>>::create_constructor(ctx);
     FunctionType list_constructor = ObjectWrap<T, ListClass<T>>::create_constructor(ctx);
-    FunctionType set_type_constructor = ObjectWrap<T, SetClass<T>>::create_constructor(ctx);
+    FunctionType set_constructor = ObjectWrap<T, SetClass<T>>::create_constructor(ctx);
+    FunctionType dictionary_constructor = ObjectWrap<T, DictionaryClass<T>>::create_constructor(ctx);
     FunctionType realm_object_constructor = ObjectWrap<T, RealmObjectClass<T>>::create_constructor(ctx);
     FunctionType results_constructor = ObjectWrap<T, ResultsClass<T>>::create_constructor(ctx);
 
     PropertyAttributes attributes = ReadOnly | DontEnum | DontDelete;
     Object::set_property(ctx, realm_constructor, "Collection", collection_constructor, attributes);
     Object::set_property(ctx, realm_constructor, "List", list_constructor, attributes);
-    Object::set_property(ctx, realm_constructor, "Set", set_type_constructor, attributes);
```

Comment:
Hopefully the "Set" constructor doesn't go null again.  :) 
The name change to `set_type_constructor` somehow made the constructor work again.  But I hope that this just works now.

Reply:
I would appear that the tests regarding the older null Set constructor have passed.  That is definitely a good thing.

Reply:
I think downloading a published version (without Set) was triggered and thereby Set was null. I have changed to the code to look like the rest.